charts/astronomer/templates/_helpers.yaml

```diff
@@ -268,6 +268,8 @@ s3:
 - name: HOUSTON_SCRIPT_UPDATE_RUNTIME_SERVICE_URL
   value: {{ .Values.houston.updateRuntimeCheck.url }}
 {{- end }}
+- name: COMMANDER_WAIT_ENABLED
+  value: "false"
```

Comment:
for dataplane mode and control plane mode it can be false
for unified mode this check is needed since commander will be running on control plane that wait for host check will ensure commander becomes active